Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/Editor/EditorSystemComponent.cpp

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Editor/EditorSystemComponent.h>
+#include <EMStudio/AtomRenderPlugin.h>
+
+#include <EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/EMStudioManager.h>
+
+namespace AZ::EMotionFXAtom
+{
+    void EditorSystemComponent::Reflect(ReflectContext* context)
+    {
+        if (SerializeContext* serialize = azrtti_cast<SerializeContext*>(context))
```

Comment:
uber nit: This is a minor style thing but with casts like this, as the type information is on the right of the expression, you can remove redundant type information by using `auto` on the left here (as the full type is on the right, no type deduction takes place) It's just omitting duplicate info.

```
if (auto serialize = azrtti_cast<SerializeContext*>(context))
```

Little bit less typing and slightly less visual noise but totally up to you